### ISSUE TITLE
Sprint 1.36.0 1/2 - May 2 to May 13

### ISSUE THREAD

**Issue by marcushyett-ph** [PostHog/posthog#9533]
# Global Sprint Planning

## Retro: Status of Outcomes from Previous Sprint

1. Unblock persons on events querying on self hosted (@yakkomajuri) (Status: On track. After Friday we will be able to toggle flags to turn on the entire persons/groups on events setup on self-hosted and run queries against the new schema.)
  - why: To speed up querying
2. Have all infrastructure ready to do clickhouse migration (@guidoiaquinti) (Status: on track - we're pivoting to upgrade after open heart surgery)
  - why: Because we're blocked on doing schema migration 
3. Cohort filtering implementation (@EDsCODE) (Status: 80%? A ton of work is finished here. Query class, UI, backwards compatibility all just about handled. Integration work needs to happen so we'll need another week)
  - why: lifecycle capabilities for cohorts are requested often as features
4. Get statsig increase in conversion from signup to first event ingested on onboarding experiment (@liyiy) (Status: support gero duty will delay implementation by a week but things are still on track as the next sprint was intended to create/run experiments/metrics anyway. onboarding 1.5 in the plans during this time)
  - why: so we can have more customers be successful with PostHog
7. Have great demo data live on demo.posthog.com (@Twixes) (Status: Early stages primarily due to Support Gero duty, will roll into the first half of the next sprint.)
  - why: to increase sign ups
9.  Complete nail insights RFC implementations https://github.com/PostHog/product-internal/pull/290 (@mariusandra) (Status: in progress, will finish latest earl next week)
  - why: m2m dashboard: customer requests

## Plan: Proposed Goals for Next Sprint 

Each goal should have a single owner. Owner can only be an engineer.

1. Have a team on Cloud successfully running the full persons/groups on events setup (@yakkomajuri)
* Why? To speed up querying for our biggest customers
1. Have great demo data live on demo.posthog.com (@Twixes)
  - why: to increase sign ups
1.  Complete nail insights RFC implementations https://github.com/PostHog/product-internal/pull/290 (@mariusandra)
  - why: m2m dashboard: customer requests
1. Frontend "Apps" Support Marketing launch(including notification banner app) (@mariusandra)
 - why: Extensibility
1. Cohort filtering implementation (@EDsCODE) 
  - why: lifecycle capabilities for cohorts are requested often as features
1. Get statsig increase in conversion from signup to first event ingested on onboarding experiment (@liyiy) 
  - why: so we can have more customers be successful with PostHog
1. Get end-to-end persons on events querying working for a single team for Trends (@EDsCODE  TBC)
 - Why? To speed up querying for our biggest customers
1. Complete the work to build an environment completely from scratch and automated (using code, no clicks on a UI). Target is to end up with Heroku (PostgreSQL + Kafka) and AWS resources (EKS with Helm Chart for PostHog) for everything else.  @guidoiaquinti @hazzadous 
- Why? It's a big step forward/milestone for our 6 months team goals.
1. Team Platform (@fuziontech ): in-place upgrade to get ClickHouse 21.11.1 in PROD (finally, hopefully!). 
- Why? We want to get CH feature/bugfix

# Team sprint planning

For your team sprint planning copy this template into a comment below for each team.

```
Team ___

## Retro

<!-- Talk about what went well, what didn't go well and any actions to improve next time -->

- 

## Hang over items from previous sprint

<!-- For each item, decide to re-prioritise (and add below) or deprioritise -->

- Item 1. prioritised/deprioritise

## Planning

<!-- Each item should have a single owner. Owner can only be an engineer. -->

### High priority

-

### Low priority / side quests

-

```

**Comment by guidoiaquinti** [comment]
> Have all infrastructure ready to do clickhouse migration (@guidoiaquinti) (Status: ❓)
> why: Because we're blocked on doing schema migration

ClickHouse schema migrations are now working again in PROD. We are going to continue with the infrastructure as code work for now but we are likely going to switch focus once we'll have the ✅ from the ingestion team to perform the in-place CH upgrade to 21.11.11.1.

**Comment by yakkomajuri** [comment]
# Team Ingestion

## Retro

- @tiina303 
    - Happy to get billing project shipped (and out of the way)
- @fuziontech
    - Really good sprint, fixed cluster issues + we understand exactly what happened
    - Happy we're engaging Altinity
- @yakkomajuri 
     - Solid sprint with some setbacks

## Hang over items from previous sprint

None.

## Planning

Have a team on Cloud successfully running the full persons/groups on events setup (@yakkomajuri)

https://github.com/orgs/PostHog/projects/41
